tedana/gscontrol.py

```diff
@@ -5,15 +5,14 @@
 import os.path as op
 
 import numpy as np
-from numpy.linalg import lstsq
 from scipy import stats
 from scipy.special import lpmv
 
 from tedana import io, utils
 
 LGR = logging.getLogger(__name__)
-RepLGR = logging.getLogger('REPORT')
-RefLGR = logging.getLogger('REFERENCES')
+RepLGR = logging.getLogger("REPORT")
```

Comment:
Was this just for style ?

Reply:
Yeah, I ran `black` and `isort` on the file and then only kept changes from the top of the file and the `minimum_image_regression` function.

Reply:
Scratch that- I don't think I ran `isort`. But I _did_ run `black`.